com.arc.embeddedcdt/src/com/arc/embeddedcdt/LaunchConfigurationConstants.java

```diff
@@ -25,6 +25,7 @@
             + ".debugger_build_before_launch";
     static final boolean ATTR_DEBUGGER_BUILD_BEFORE_LAUNCH_DEFAULT = true;
 
+    String ATTR_DEBUGGER_PLUGIN_VERSION = LAUNCH_ID + ".debugger_plugin_version";
```

Comment:
Better to have same separate name, like `FILE_FORMAT` or similar, because plugin version is something that is tied to release, but this number represents file format, which may not change between plugins versions.